src/esn.calendar.libs/app/components/time-select/time-select-scroll.directive.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+angular.module('esn.calendar.libs')
+  .directive('timeSelectScroll', timeSelectScroll);
+
+function timeSelectScroll() {
+  return {
+    restrict: 'A',
+    link: function($scope, element) {
+
+      $scope.$on('$mdMenuOpen', function() {
+        const parent = angular.element(element[0]);
+        const target = angular.element(element[0].querySelector('.active-item'));
```

Comment:
Have you tried `target.scrollIntoView()`?

Reply:
yes, doesn't work

Reply:
now it does, thanks.